### PR TITLE
Check fonts files with phpstan too

### PR DESCRIPTION
**Pull request by mvorisek**
https://github.com/phpstan/phpstan/issues/7955 was fixed

### CODE REVIEW

**Review by williamdes — DISMISSED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -132,4 +132,4 @@ jobs:
             - name: Install phpstan
               run: composer require --dev phpstan/phpstan
             - name: Analyse files
-              run: ./vendor/bin/phpstan --memory-limit=2G
+              run: ./vendor/bin/phpstan --memory-limit=-1
```
Comment (williamdes) [inline .github/workflows/tests.yml:135]: Thats kind of evil ^^
Maybe @staabm would like to debug this phpstan memory issue?

Comment (staabm) [reply .github/workflows/tests.yml:135]: > when parallel processing is enabled (default)

yeah, thats expected. every worker proces has the limit you define here. so e.g. with 4 workers you allow phpstan to consume 4x memory-limit to consume

Comment (staabm) [reply .github/workflows/tests.yml]: btw: I am fine with analyzing stuff, but just running phpstan on this project already shows me that alone analyzing the main file takes 7 minutes.

```
/Users/staabm/workspace/TCPDF/tcpdf.php
--- consumed 390 MB, total 930 MB, took 442.72 s
```

@mvorisek I would have expected that someone with interesst in this TCPDF project would have at least analyzed the slow files or similar. expecting that I do that all for you is like a bad joke.

just telling some random open source developer spending their freetime on your problems is not the way how this works

Comment (williamdes) [reply .github/workflows/tests.yml:135]: > expecting that I do that all for you is like a bad joke.
just telling some random open source developer spending their freetime on your problems is not the way how this works

I apologize for the ping then, I should have looked at it closer before

Comment (staabm) [reply .github/workflows/tests.yml:135]: fine then. I think analyzing this project could be way faster by splitting the huge files into parts and getting rid of the manual require/include calls but use regular autoloading instead.

Comment (mvorisek) [reply .github/workflows/tests.yml:135]: we need to wait for 1.9.5 phpstan release, then let's check the performance and memory requirements again

Comment (staabm) [reply .github/workflows/tests.yml:135]: fwiw, I did my measure with 1.9.x-dev

**Review by williamdes — DISMISSED** [review]

**Review by nicolaasuni — APPROVED** [review]

**Pull request merged**